code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -78,7 +78,7 @@
 			qdel(src)
 			return
 		if(is_pen(W))
-			var/decalselection = input("Please select a decal") as null|anything in list("Atmospherics", "Bartender", "Barber", "Blueshield", "Captain",
```

Comment:
You don't actually remove the decal from the cardboard dmi, please either return this or remove it from the dmi also (imo i think it would be fine to leave it, we're just removing the job, not everything associated with it)

Reply:
Returned, so should be done!